build.sbt

```diff
@@ -0,0 +1,13 @@
+scalaVersion := "2.12.8"
+
+name := "Dank Scala Project"
+
+libraryDependencies += "org.apache.spark" %% "spark-sql" % "2.4.1"
```

Comment:
2.3.0, gotta have that spark on kube integration

Reply:
koo ber nay tayce